state_machine/applications/flight/lib/logs.py

```diff
@@ -31,12 +31,16 @@ def beacon_packet():
     mag = cubesat.magnetic if cubesat.imu else array([nan, nan, nan])
     rssi = cubesat.radio.last_rssi if cubesat.radio else nan
     fei = cubesat.radio.frequency_error if cubesat.radio else nan
+    sun = cubesat.sun_vector if (
```

Comment:
We probably want to log the pure sun sensor data rather than the sun_vector.
We will probably remove the sun vector function, it's kinda flawed.

Reply:
sounds good, I changed it so that it pulls the lux value from each individual sun sensor. I also added functions within the drivers that enable you to get either the positive or negative sun sensor values as an (x, y, z) array if needed.

Reply:
This also increases the beacon packet size to 76 bytes as there are 6 new floats, each being the lux value of a sun sensor.